nautobot/virtualization/forms.py

```diff
@@ -469,6 +469,14 @@ class VirtualMachineFilterForm(
 
 
 class VMInterfaceForm(NautobotModelForm, InterfaceCommonForm):
+    parent_interface = DynamicModelChoiceField(
+        queryset=VMInterface.objects.all(), required=False, label="Parent interface"
+    )
+    bridge = DynamicModelChoiceField(
+        queryset=VMInterface.objects.all(),
+        required=False,
+        label="Bridged interface",
+    )
```

Comment:
Please add `help_text` to these fields since it won't be inherited from the model as they were overloaded on this form.